src/ds5/ds5-active.cpp

```diff
@@ -84,7 +84,7 @@ namespace librealsense
 
             // EMITTER FREQUENCY OPTION
             if( ( _pid == ds::RS457_PID || _pid == ds::RS455_PID )
-                && _fw_version >= firmware_version( "5.13.1.53" ) )
+                && _fw_version >= firmware_version( "5.14.0" ) )
```

Comment:
Didn't you mean 5.14.0.0 ?

Reply:
It's the same,
The utility `version` class drops the build number if it's 0.
I tested it.

Reply:
if so ... :-)